lms/static/js/student_account/views/FormView.js

```diff
@@ -119,7 +137,7 @@
 
             escapeStrings: function( obj ) {
                 _.each( obj, function( val, key ) {
-                    obj[key] = _.escape( val );
+                    obj[key] =  HtmlUtils.HTML( val );
```

Comment:
I'm not sure about this. This is now marking the values as being already escaped, so the strings will not be escaped when rendered. This mean that the function's name is incorrect (it isn't doing any escaping). 

My temptation is to remove this method entirely. The intention is that escaping happens nearer the usage, which in this case is in the Underscore template itself.